### ISSUE TITLE
KE: Member of The National Assembly at an Organisation other than Parliament

### ISSUE THREAD

**Issue by tmtmtmtm** [mysociety/pombola#2065]
```
  id   |       legal_name       |   organisation    |              role               
-------+------------------------+-------------------+---------------------------------
   194 | Oburu Ngona Odinga     | REPUBLIC OF KENYA | Member of the National Assembly
   382 | Amina Ali Abdalla      | REPUBLIC OF KENYA | Member of the National Assembly
  3012 | Daniel Wanyama Sitati  | REPUBLIC OF KENYA | Member of the National Assembly
 13128 | Patrick Wangamati      | REPUBLIC OF KENYA | Member of the National Assembly
 13129 | Isaac Maigua Mwaura    | REPUBLIC OF KENYA | Member of the National Assembly
 13130 | Zulekha Juma Hassan    | REPUBLIC OF KENYA | Member of the National Assembly
 13131 | Johnson Arthur Sakaja  | REPUBLIC OF KENYA | Member of the National Assembly
 13132 | Janet Marania Teiyaa   | REPUBLIC OF KENYA | Member of the National Assembly
 13133 | Hassan Aden Osman      | REPUBLIC OF KENYA | Member of the National Assembly
 13134 | Sara Paulata Korere    | REPUBLIC OF KENYA | Member of the National Assembly
 13135 | Abdi Noor Mohammed Ali | REPUBLIC OF KENYA | Member of the National Assembly
 13136 | Sunjeev Kour Birdi     | REPUBLIC OF KENYA | Member of the National Assembly
 13137 | Robert Mutemi Mutua    | REPUBLIC OF KENYA | Member of the National Assembly
 13152 | Fatuma Ibrahim Ali     | REPUBLIC OF KENYA | Member of the National Assembly
(14 rows)
```

via

```
SELECT mp.id, mp.legal_name, o.name AS organisation, t.name AS role
  FROM core_person mp
  LEFT JOIN core_position p ON mp.id = p.person_id
  LEFT JOIN core_positiontitle t ON p.title_id = t.id
  LEFT JOIN core_organisation o ON p.organisation_id = o.id
 WHERE t.name ILIKE '%Member of the National Assembly%'
   AND o.name != 'Parliament'
 ORDER BY t.name, o.name, mp.id
```

**Comment by tmtmtmtm** [comment]
This should probably be fixed before #1935, though the query will need updated after that.

See also #2064 for the same problem for 'Member of Parliament'

**Comment by mhl** [comment]
The current examples of this problem have been fixed, and this check is now being done every night in `kenya_check_for_recurring_problems`.
